umbraco-cms/tutorials/creating-a-custom-dashboard.md

```diff
@@ -660,6 +659,155 @@ angular.module("umbraco").controller("CustomWelcomeDashboardController", functio
 </div>
 ```
 
+## Extending the Dashboard using the UI library
+
+Now that we have a fully functioning dashboard we might want to make it prettier.
+
+To do this we can use the Umbraco UI library.
+
+The [Umbraco UI Library](../extending/ui-library.md) is a set of web components that we can use to build Umbraco User Interfaces.
+
+To get started using the UI library, the easiest way is to add the following Content Delivery Network (CDN) script to the **WelcomeDashboard.html:** file.
+
+```javascript
+<script src="https://cdn.jsdelivr.net/npm/@umbraco-ui/uui@latest/dist/uui.min.js"></script>
```

Comment:
Will it make sense to add the location where this line should be added? For eg: at the start or end of the file? I had to scroll down to the final code to see where to add this line 🙂

Reply:
Yeah that makes good sense :D